grammar/coqpp_main.ml

```diff
@@ -260,22 +262,77 @@ let print_ast chan ext =
 
 end
 
+module TacticExt :
+sig
+
+val print_ast : Format.formatter -> tactic_ext -> unit
+
+end =
+struct
+
+let rec print_symbol chan = function
```

Comment:
`chan` here is misleading, it should be `fmt`

Reply:
I used chan all over the place already, this is just me being consistent with the code.

Reply:
Yeah so maybe a quick search/replace would help; this will greatly mislead people IMVHO.

Reply:
Greatly? Don't you think you're exaggerating a bit? But yeah, let's search-and-replace.

Reply:
That's really changing like every line in `coqpp_main`. I don't think it's very good from the point of view of readability of the diff. Should I do that in another commit at least?

Reply:
Oh sure do as you wish, if you think this is bad we can may don't do it; but I see the confusion between the two `fprintf` often enough as for this to worry me.